common/meter.hpp

```diff
@@ -15,5 +15,5 @@ class Meter {
   const std::string dwarf_name_;
   MeasureResults &result_;
   DwarfParams params_;
-  RunOptions opts_;
+  RunOptions const *opts_;
```

Comment:
What's this for?

Reply:
I replaced this with a pointer in order to use polymorphism for `RunOptions`.
It will not be possible to store the reference here, since it must be initialized in the constructor, and our logic suggests otherwise.

Reply:
Also with C-pointers it looks a bit strange
For a more beautiful (as it seems to me) solution, a little more work needs to be done, I have already started working on it [here](https://github.com/bagrorg/dwarf_bench/pull/39)

Reply:
Do you want to introduce it with this PR or separate?

Reply:
Separately, I think there is still a lot to change